src/maud/posterior_predictive_model.stan

```diff
@@ -0,0 +1,371 @@
+functions{
+#include functions.stan
+  vector get_flux(vector conc,
```

Comment:
If this function and `dbalanced_dt` is the same as the `get_flux` in `model.stan` then they should probably be moved (I guess to `functions.stan`) and included in both places. Otherwise the two functions will likely diverge over time.